todo/src/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.scss';
+import App from './App';
+import {Provider} from 'react-redux';
+import {createStore} from 'redux';
+import rootReducer from './reducers';
+
+
+ReactDOM.render(
+    <Provider store={createStore(rootReducer, 
+        window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__())}>
```

Comment:
Is there a reason you are adding the createStore in line here, but extracting the mapDispatchToProps in your other file?